src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsTransport.cs

```diff
@@ -40,11 +41,13 @@ public ServerSentEventsTransport(HttpClient httpClient, ILoggerFactory loggerFac
             _logger = (loggerFactory ?? NullLoggerFactory.Instance).CreateLogger<ServerSentEventsTransport>();
         }
 
-        public Task StartAsync(Uri url, Channel<byte[], SendMessage> application)
+        public Task StartAsync(Uri url, Channel<byte[], SendMessage> application, TransferMode requestedTransferMode)
         {
             _logger.LogInformation("Starting {transportName}", nameof(ServerSentEventsTransport));
 
             _application = application;
+            Mode = TransferMode.Text; // SSE transport supports only text mode
```

Comment:
The actual mode can be different from the requested mode which tells the connection it needs to convert data